eng/pipelines/templates/variables/globals.yml

```diff
@@ -17,3 +17,7 @@ variables:
   DOTNET_NOLOGO: 1
   Package.EnableSBOMSigning: true
   Codeql.SkipTaskAutoInjection: true
+  # Define optional AdditionalTestFilters via pipeline runtime variables or matrix config variables
+  # Variable value must start with either & or | to join conditions.
```

Comment:
This may be a bit obscure and difficult to trip over if the need arises.   Any chance that we've got a wiki or something with pipeline variables that we could add this to?

Reply:
Yeah I've been trying to think of a good discoverable doc location, if you have ideas. I can create language specific test config docs underneath the broader https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/389/Configuring-Tests doc? The storage virtualized matrix will be adding some example usage of this variable as well soon.

Ideally you wouldn't have to know to prefix with an operator, but I haven't thought of a clean way to chain statements yet either without possibly breaking out into a script and doing some more parsing. But that adds complexity of a different nature as well.

Reply:
I'm not sure if this is the best approach but thought I would put it up in case anyone else had alternative ideas.

Reply:
> Yeah I've been trying to think of a good discoverable doc location, if you have ideas. 

Unfortunately, this is one thing that we don't seem to have... but the DevOps wiki seems to align well with most of the EngSys content, so I'd agree that it's probably the best place.

Reply:
Ok I added some preliminary docs [here](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/742/azure-sdk-for-net). As we get more usage examples I'll link them in the docs. I'm hesitant to include a link to the internal docs in the source code though, unless you think I should?

Reply:
No, I agree... probably not.